samples/Simple.HttpPatch.Samples/Program.cs

```diff
@@ -7,8 +7,6 @@
 
 var app = builder.Build();
 
-app.UseHttpsRedirection();
```

Comment:
what is the reason of removing this?

Reply:
Because I deleted it locally for debugging, I forgot to restore